index.ts

```diff
@@ -613,8 +613,12 @@ function unwrapPackageName(obj: any, packageName: string): fixturify.DirJSON {
   return getFolder(obj, packageName);
 }
 
+function isError(e: unknown): e is Error {
+  return !!e && typeof e === 'object' && Object.prototype.toString.call(e) === '[object Error]' && 'message' in e;
```

Comment:
Let's simplify this further, `isErrnoException` doesn't really provide much value, we basically just want to ensure two things in this guard:

1. e is an object
2. e is not null

As the conditional clauses this guards manually check `e.code` for specific error codes

Reply:
Already done. See latest pushes!